flowauth/frontend/src/ServerAdminDetails.jsx

```diff
@@ -167,24 +168,27 @@ class ServerAdminDetails extends React.Component {
     const { item_id } = this.props;
     if (item_id !== -1) {
       try {
-        const capAwait = getCapabilities(item_id);
+        const capAwait = getScopes(item_id);
         const limitsAwait = getTimeLimits(item_id);
-        const scopeGraph = scopesGraph(await capAwait);
+        const rights = await capAwait.then(
+            resp => resp.reduce((p_x,x) => ({[x.name]:x.enabled, ...p_x}), {})
+          );
+        console.log("Rights", rights)
+        const scope_graph = scopesGraph(rights);
+        console.log("scopeGraph", scope_graph);
         const enabledKeys = [];
-        const rights = await capAwait;
-        const scopes = jsonify(
-          scopeGraph,
-          [],
-          Object.keys(rights).filter((k) => rights[k]),
-          enabledKeys
-        );
+        const scopes = jsonify(scope_graph, enabledKeys);
         const serverName = (await getServer(item_id)).name;
         const latestExpiry = (await limitsAwait).latest_token_expiry;
-        const maxLife = (await limitsAwait).longest_token_life;
+        const maxLife = (await limitsAwait).longest_token_life_minutes;
+        console.log(await limitsAwait)
```

Comment:
That's curious - I haven't see await in console.log before.
Are you sure this is required?

Reply:
I think if something's already been `await`'d, the result should just be returned as aif it was a normal variable - I guess strictly it doesn't need to be there, though